plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClientModule.java

```diff
@@ -78,6 +80,7 @@ public static ConnectionFactory connectionFactory(BaseJdbcConfig config, Credent
                 new OracleDriver(),
                 config.getConnectionUrl(),
                 connectionProperties,
-                credentialProvider));
+                credentialProvider),
+                exception -> exception instanceof SQLRecoverableException);
```

Comment:
Oracle connector always uses `RetryingConnectionFactory` regardless of the configuration for backward compatibility.